src/test/resources/dlsfls/roles_tlq.yml

```diff
@@ -0,0 +1,18 @@
+---
+_meta:
+  type: "roles"
+  config_version: 2
+  
+os_dls_tlq_lookup:
+  cluster_permissions:
+    - "*"
+  index_permissions:
+    - index_patterns:
+      - "tlqdocuments"
+      dls: "{ \"terms\": { \"access_codes\": { \"index\": \"user_access_codes\", \"id\": \"${user.name}\", \"path\": \"access_codes\" } } }"
```

Comment:
What does `path` mean in `dls`? There is no such field in the document.

Reply:
that `path` is part of the `terms` query in the DLS. namely, this is a terms lookup query.
sadly, the [opensearch docs for terms queries](https://opensearch.org/docs/latest/opensearch/query-dsl/term/#terms) do not document this at all (i've raised [documentation-website#430](https://github.com/opensearch-project/documentation-website/issues/430) for this), but it's documented in the [corresponding ES docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/query-dsl-terms-query.html#query-dsl-terms-lookup).

this is actually the feature for which this whole PR has been built :)